### ISSUE TITLE
Is the matching engine distributed?

### ISSUE THREAD

**Issue by anooshcnayak** [gitbitex/gitbitex-spot#25]
If I have a multi machine deployment? Will the matching happen across the machines? I could see the order getting stored in-memory. Will this not bloat up the memory if I have thousands of order across multiple products?

**Comment by ammanpashasc** [comment]
Hey, were you able to figure this out?

**Comment by anooshcnayak** [comment]
@ammanpashasc  Yes! it is not distributed. Single threaded with orderbook snapshots taken in regular intervals & stored in redis
